from .myException import NoPositiveError as NoPositiveError


